modules/spo-runtime-metrics.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-advanced.adoc
+
+:_content-type: REFERENCE
+[id="spo-runtime-metrics_{context}"]
+= controller-runtime metrics
+
+The controller-runtime `metrics` and the DaemonSet endpoint `metrics-spod` provide a set of default metrics. Additional metrics are provided by the daemon, which are always prefixed with `openshift_security_profiles_`.
```

Comment:
They're prefixed by `security_profiles_operator_`
(Sorry, my fault again, this is another namespace that we keep as-is from upstream)